src/web/client/extension.ts

```diff
@@ -308,6 +308,12 @@ export function processWorkspaceStateChanges(context: vscode.ExtensionContext) {
                     if (entityInfo.entityId && entityInfo.entityName) {
                         context.workspaceState.update(document.uri.fsPath, entityInfo);
                         WebExtensionContext.updateVscodeWorkspaceState(document.uri.fsPath, entityInfo);
+
+                        if (isCoPresenceEnabled() && tab.input instanceof vscode.TabInputCustom) {
```

Comment:
not the bext place to handle these changes - but to unblock this feature - lets get this in. please do create a task for refactor.

Reply:
sure!